src/Classes/PassiveSpec.lua

```diff
@@ -301,7 +301,7 @@ function PassiveSpecClass:EncodeURL(prefix)
 	local masteryNodeIds = {}
 
 	for id, node in pairs(self.allocNodes) do
-		if node.type ~= "ClassStart" and node.type ~= "AscendClassStart" and id < 65536 then
+		if node.type ~= "ClassStart" and node.type ~= "AscendClassStart" and id < 65536 and nodeCount < 255 then
```

Comment:
cant you do this check before the for loop?

Reply:
nodeCount is incremented as they are added so this is what this is handling. Might be able to do a check on the size of allocNodes before but then you just wouldn't have any outputted tree.

Reply:
ahh k, yeah then this is fine